### DISCUSSION TITLE
[YetiForcePDF] Setting Author and making document read-only

### DISCUSSION THREAD

**Discussion by diderich** [YetiForceCompany/YetiForceCRM#16608] [How to]
Is there a possibility to set the Author in the metadata when creating a PDF file using the YetiForcePDF library (not related to the CRM system)?
Is there a possibility to set the document read-only so that the user cannot change its content using the YetiForcePDF library (not related to the CRM system)?

**Comment by rskrzypczak** [comment]
https://github.com/YetiForceCompany/YetiForcePDF/blob/ca04b8c0ef97592e1d2fbc77264dbd5323b6e4c6/lib/Objects/Meta.php#L53
https://github.com/YetiForceCompany/YetiForceCRM/blob/33aca526ea3899fa8bab576624ab94dedfa1a732/app/Pdf/Drivers/YetiForcePDF.php#L131

> Is there a possibility to set the document read-only so that the user cannot change its content using the YetiForcePDF library (not related to the CRM system)?

As far as I know, this is not possible at this moment.

**Comment by mariuszkrzaczkowski** [comment]
Did you see it?
![image](https://user-images.githubusercontent.com/10198654/197459488-e2641c83-bc71-4b6e-bf29-245b2e2870fa.png)

**Reply by diderich** [reply]
I am using the PDF library in another project independent of the CRN system 😉

**Reply by mariuszkrzaczkowski** [reply]
I understand now, check it out
https://github.com/YetiForceCompany/YetiForceCRM/blob/434d2048c12910b6e1762192665ec913aef625f8/app/Pdf/Drivers/YetiForcePDF.php#L128-L132
